src/include/c_api/kuzu.h

```diff
@@ -4,6 +4,8 @@
 #include <stdlib.h>
 #include <string.h>
 
+#include <common/arrow/arrow.h>
```

Comment:
This is going to require some packaging changes so that it gets included in the released C API header. Either combining them at release time like with the C++ headers, or maybe just inlined here.